### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 10: ordinal not in range(128)

### ISSUE THREAD

**Issue by willkg** [willkg/dennis#96]
From #94:

Using Fedora 25 and French language:

```
$ .local/bin/dennis-cmd lint file.po 
dennis version 0.7
Oh no! Dennis has thrown an error while trying to do stuff.
Please write up a bug report with the specifics so that 
we can fix it.

https://github.com/willkg/dennis/issues

Here is some information you can copy and paste into the 
bug report:

---
Dennis: '0.7'
Python: '2.7.12 (default, Sep 29 2016, 12:52:02) \n[GCC 6.2.1 20160916 (Red Hat 6.2.1-2)]'
Command line: ['.local/bin/dennis-cmd', 'lint', 'T\xc3\xa9l\xc3\xa9chargements/dnf.po']
Traceback (most recent call last):
  File ".local/bin/dennis-cmd", line 11, in <module>
    sys.exit(click_run())
  File "/home/jb/.local/lib/python2.7/site-packages/dennis/cmdline.py", line 156, in click_run
    cli(obj={})
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 991, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/home/jb/.local/lib/python2.7/site-packages/dennis/cmdline.py", line 282, in lint
    '>>> Working on: {fn}'.format(fn=fn),
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 10: ordinal not in range(128)
```

**Comment by willkg** [comment]
I can't seem to reproduce this.

Amongst other things, the suggested command line doesn't match the one listed in the bug report information, so it seems like something was changed and the steps to reproduce aren't correct. I think the problem I'm hitting is related to that somehow.

@Jibec Can you give me steps to reproduce and/or a test case?

**Comment by Jibec** [comment]
sorry for late answer! I did:

```
[jb@fixe Téléchargements]$ pip install --user dennis blessings --upgrade
Requirement already up-to-date: dennis in /home/jb/.local/lib/python2.7/site-packages
Requirement already up-to-date: blessings in /home/jb/.local/lib/python2.7/site-packages
Requirement already up-to-date: polib>=1.0.6 in /home/jb/.local/lib/python2.7/site-packages (from dennis)
Requirement already up-to-date: click==6.6 in /home/jb/.local/lib/python2.7/site-packages (from dennis)
You are using pip version 8.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command
```

then:
```
[jb@fixe Téléchargements]$ pwd
/home/jb/Téléchargements
```

```
[jb@fixe Téléchargements]$ ~/.local/bin/dennis-cmd lint ./messages-ttrss.po 
dennis version 0.8
Oh no! Dennis has thrown an error while trying to do stuff.
Please write up a bug report with the specifics so that 
we can fix it.

https://github.com/willkg/dennis/issues

Here is some information you can copy and paste into the 
bug report:

---
Dennis: '0.8'
Python: '2.7.12 (default, Sep 29 2016, 12:52:02) \n[GCC 6.2.1 20160916 (Red Hat 6.2.1-2)]'
Command line: ['/home/jb/.local/bin/dennis-cmd', 'lint', './messages-ttrss.po']
Traceback (most recent call last):
  File "/home/jb/.local/bin/dennis-cmd", line 11, in <module>
    sys.exit(click_run())
  File "/home/jb/.local/lib/python2.7/site-packages/dennis/cmdline.py", line 156, in click_run
    cli(obj={})
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/jb/.local/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/home/jb/.local/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/jb/.local/lib/python2.7/site-packages/dennis/cmdline.py", line 282, in lint
    '>>> Working on: {fn}'.format(fn=fn),
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 10: ordinal not in range(128)

---
```

Same command in /home/jb do not crash.

**Comment by willkg** [comment]
Ahh... ok. I think I see what I'm doing wrong. I'll try to get a test failing next week and fix it then.

**Comment by Jibec** [comment]
kind remember: any update here?

**Comment by willkg** [comment]
@Jibec sorry about that--should be fixed now. I'll spend some time looking around for other things and then do a release this week some time.

**Comment by willkg** [comment]
Released in v0.9. Hope that helps!
